cfpb/consumerfinance.gov
# Retirement: Hide Tool - Spanish Translation

This is a follow-up on #7929 - a warning/alert field on the [before-you-claim](https://www.consumerfinance.gov/consumer-tools/retirement/before-you-claim/) page

**To test:** 

- navigate to: http://localhost:8000/consumer-tools/retirement/before-you-claim
- disable javascript
- reload page and verify warning field is shown and retirement tool is hidden
- click "Ver página en español"
- verify the text has been translated to Spanish
- text should read: "Este sitio no es compatible con su navegador web. Intente con un navegador web más nuevo o confirme que JavaScript está habilitado."

![Screenshot 2023-08-18 at 5 04 07 PM](https://github.com/cfpb/consumerfinance.gov/assets/130792942/34da1e37-8344-46d4-b794-0b8ed65a076c)
